src/server/zset_family.cc

```diff
@@ -34,6 +36,7 @@ static const char kNxXxErr[] = "XX and NX options at the same time are not compa
 static const char kScoreNaN[] = "resulting score is not a number (NaN)";
 static const char kFloatRangeErr[] = "min or max is not a float";
 static const char kLexRangeErr[] = "min or max not valid string range item";
+const char GeoAlphabet[] = "0123456789bcdefghjkmnpqrstuvwxyz";
```

Comment:
maybe `constexpr` and rename to `kGeoAlphabet`?

Reply:
Reading the code below - maybe use `constexpr string_view` so that you have `.size()`, and you can add `static_assert()` on the size and use it as modulo for accessing (see below)